### ISSUE TITLE
widened Z-disks 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5686]
**Preferred term label:**

widened Z-disks 
**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/23746549/

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0020202

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
